db/goleveldb.go

```diff
@@ -44,6 +44,7 @@ func LoadGoLevelDB(name, dir string, cacheSizeMeg int, bufferSizeMeg int, collec
 	}
 
 	if collectMetrics {
+		prometheus.Unregister(metrics.NewStatsCollector(fmt.Sprintf("goleveldb_%s", name), log.Default, db)) // unregister previous registered
```

Comment:
If this was added because `Register` returned an error then we should probably fix the error handling to check for `AlreadyRegisteredError` being returned from `Register` and ignore it.

Reply:
Since we don't need to reload evm.db again this line is not necessary now.